cmd/notary/main.go

```diff
@@ -11,20 +10,25 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
+	"github.com/docker/docker/pkg/homedir"
 	"github.com/docker/notary/pkg/passphrase"
 	"github.com/docker/notary/version"
 )
 
-const configFileName string = "config"
-const defaultTrustDir string = ".notary/"
-const defaultServerURL = "https://notary-server:4443"
-const idSize = 64
+const (
+	configFileName   string = "config"
+	defaultTrustDir  string = ".notary/"
+	defaultServerURL        = "https://notary.docker.io"
```

Comment:
Note that there is no current public notary server. We would have to deal with GUN ownership for that to be a reality.